code/modules/mod/mod_core.dm

```diff
@@ -79,9 +79,14 @@
 	/// Installed cell.
 	var/obj/item/stock_parts/cell/cell
 
+/obj/item/mod/core/standard/handle_atom_del(atom/deleting_atom)
```

Comment:
this will get handled by the cell deleting moving it to nullspace

Reply:
Huh? This is incase a badmin deletes the cell.

Reply:
when the cell gets deleted, deletion moves it to nullspace. this counts as changing location, so the exited signal would trigger, properly severing the connection